html/partials/os.html.haml

```diff
@@ -0,0 +1,25 @@
+- if os
+  %div
+    %a.scope_anchor{:id => "os"}
+    #os_container.scope
+      .row
+        .col-xs-1
+          %img.scope_logo_big{:src => "assets/logo-os.png", :class=>"over", :"data-toggle"=>"popover", :title=>"Operating System", :"data-content"=>"#{scope_help('os')}"}/
```

Comment:
Line is too long. [171/80]<br>Avoid defining `class` in attributes hash for static class names<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace